src/Microsoft.ApplicationInsights/Extensibility/Implementation/Experimental/ExperimentalFeaturesExtension.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Experimental
+{
+    using System;
+    using System.ComponentModel;
+    using System.Linq;
+
+    /// <summary>
+    /// This class provides a means to interact with the <see cref="TelemetryConfiguration.ExperimentalFeatures" />.
```

Comment:
Watch your language, cap! :) It's no longer an array, I guess, at least as per tests above, I haven't got to the chapter 10 yet, though to prove anything...
P.S. And yes, it had to be a separate comment to spam others after all they did to my innocent inbox!